pkgs/build-support/go/module.nix

```diff
@@ -145,6 +145,11 @@ let
       cp -r --reflink=auto vendor $out
     ''}
 
+      if ! [ "$(ls -A $out)" ]; then
+        echo "vendor folder is not needed, please set 'vendorHash = null;' or 'vendorSha256 = null;' in your expression"
```

Comment:
```suggestion
        echo "vendor folder is empty, please set 'vendorHash = null;' or 'vendorSha256 = null;' in your expression"
```

Reply:
But I'm actually unsure if it's an improvement, I like yours as well.

Reply:
I'll switch to empty as it is more consistent with the other vendor error messages.